build_script.py

```diff
@@ -26,6 +26,14 @@ def run(command):
     subprocess.check_call(command, shell=True)
 
 
+def _mkdirp(path):
```

Comment:
I'm curious about this `_` and the ones on `_build` etc. Is this a Python convention I should be familiar with for internal (/non-exported?) functions? If that's the case, how is this function different from e.g. `note` in its role in the module?